### DISCUSSION TITLE
nvimtree doesn't show target folder

### DISCUSSION THREAD

**Discussion by liketoeatcheese** [nvim-tree/nvim-tree.lua#2063] [Q&A] [answered]
### Description

Hi team!

I'm currently having an issue when nvim tree doesn't show any `target` folder:

So this is the result of `ls`:
<img width="413" alt="image" src="https://user-images.githubusercontent.com/88610355/225814870-17474944-49c6-4f02-a0bd-6438601ebafd.png">

And this is nvim tree:
![image](https://user-images.githubusercontent.com/88610355/225814987-faa76b1f-4cd7-4c5a-adda-2a159b598c45.png)

### Neovim version

```text
NVIM v0.8.3
Build type: RelWithDebInfo
LuaJIT 2.1.0-beta3
```

### Operating system and version

Windows 10 wsl2

### nvim-tree version

59e65d8

### Minimal config

```lua
You can use the base config:

vim.cmd [[set runtimepath=$VIMRUNTIME]]
vim.cmd [[set packpath=/tmp/nvt-min/site]]
local package_root = "/tmp/nvt-min/site/pack"
local install_path = package_root .. "/packer/start/packer.nvim"
local function load_plugins()
  require("packer").startup {
    {
      "wbthomason/packer.nvim",
      "nvim-tree/nvim-tree.lua",
      "nvim-tree/nvim-web-devicons",
      -- ADD PLUGINS THAT ARE _NECESSARY_ FOR REPRODUCING THE ISSUE
    },
    config = {
      package_root = package_root,
      compile_path = install_path .. "/plugin/packer_compiled.lua",
      display = { non_interactive = true },
    },
  }
end
if vim.fn.isdirectory(install_path) == 0 then
  print "Installing nvim-tree and dependencies."
  vim.fn.system { "git", "clone", "--depth=1", "https://github.com/wbthomason/packer.nvim", install_path }
end
load_plugins()
require("packer").sync()
vim.cmd [[autocmd User PackerComplete ++once echo "Ready!" | lua setup()]]
vim.opt.termguicolors = true
vim.opt.cursorline = true

-- MODIFY NVIM-TREE SETTINGS THAT ARE _NECESSARY_ FOR REPRODUCING THE ISSUE
_G.setup = function()
  require("nvim-tree").setup {}
end
```

### Steps to reproduce

1. Ensure cargo is installed
2. cargo new testPathNvimTree
This will create a blank testPathNvimTree folder with related files
3. cd testPathNvimTree
4. cargo doc --open
If it opens the file in your browser just close it. We don't need it
5. back to testPathNvimTree, do `ls`, you will see the target folder and its content there.
6. nvim -nu /tmp/nvt-min.lua .
7. :NvimTreeToggle
8. You'll see that the folder is not there

### Expected behavior

NvimTree to show `target` folder

### Actual behavior

Can't see the folder

**Comment by gegoune** [comment, marked as answer]
Check if it's not being git ignored; might be something `cargo new` sets up for you.

**Reply by DougAnderson444** [reply]
Does nvimtree not show files that are in `.gitignore`? Is there a way to override this?

**Reply by DougAnderson444** [reply]
For those who landed here via search:

```lua
-- show ignored files in the tree:
opts.git.ignore = false
```

**Reply by allworldg** [reply]
https://github.com/nvim-tree/nvim-tree.lua/blob/64e2192f5250796aa4a7f33c6ad888515af50640/doc/nvim-tree-lua.txt#L557

**Comment by liketoeatcheese** [comment]
Oh yeaaaa, that's the reason. Thanks heaps!!!
